projects/daq3/a10gx/system_constr.sdc

```diff
@@ -1,28 +1,10 @@
 
-create_clock -period "10.000 ns"  -name sys_clk_100mhz      [get_ports {sys_clk}]
-create_clock -period  "1.621 ns"  -name rx_ref_clk          [get_ports {rx_ref_clk}]
-create_clock -period  "1.621 ns"  -name tx_ref_clk          [get_ports {tx_ref_clk}]
-
```

Comment:
Why were these constraints deleted for A10GX project, but they remain in the A10SOC ?

Reply:
This was a mistake, fixed it.